src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -424,6 +424,33 @@ public virtual JsonResult Json(object data, [NotNull] JsonSerializerSettings ser
         }
 
         /// <summary>
+        /// Creates a <see cref="HttpOkResult"/> object that produces an empty OK (200) response.
+        /// </summary>
+        /// <returns>The created <see cref="HttpOkResult"/> for the response.</returns>
+        [NonAction]
+        public virtual HttpOkResult Ok()
```

Comment:
No real _MVC_ compat issue for this particular method; it existed only in Web API 2.x. There, the method was named `Ok()` and the results `OkResult` and `OkNegotiatedContentResult<T>`. Doesn't seem worthwhile to maintain the generic nature of the second class nor the class names.

So I also lean toward sticking with the names I've got here.